### PR TITLE
[IOTDB-4434] Migrate SessionAlignedXXIT

### PR DESCRIPTION
**Pull request by MiniSho**
Migrate SessionAlignedABDeviceIT, SessionAlignedAggreationIT, SessionAlignedAggreationWithUnSeqIT and SessionComplexIT

### CODE REVIEW

**Review by Beyyes — COMMENTED** [review]
only migrate write correlative it tests from `integration` to `integration-test`, to reduce the cost time

File: integration-test/src/test/java/org/apache/iotdb/session/IoTDBSessionAlignedABDeviceIT.java [hunk 1]
```diff
@@ -18,56 +18,115 @@
...
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
 
 import java.util.ArrayList;
 import java.util.List;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.fail;
 
-public class IoTDBSessionVectorABDeviceIT {
```
Comment (Beyyes) [inline integration-test/src/test/java/org/apache/iotdb/session/IoTDBSessionAlignedABDeviceIT.java:42]: add comments about of the full name of `IoTDBSessionVectorABDeviceIT `

Comment (MiniSho) [reply integration-test/src/test/java/org/apache/iotdb/session/IoTDBSessionAlignedABDeviceIT.java]: done

**Review by Beyyes — APPROVED** [review]
LGTM

**Pull request merged**